rpcs3/rpcs3qt/settings_dialog.cpp

```diff
@@ -1333,6 +1334,10 @@ settings_dialog::settings_dialog(std::shared_ptr<gui_settings> gui_settings, std
 			m_gui_settings->SetValue(gui::gs_showMouseFs, val);
 		});
 
+		ui->gs_lockMouseInFullscreen->setChecked(m_gui_settings->GetValue(gui::gs_lockMouseFs).toBool());
+		connect(ui->gs_lockMouseInFullscreen, &QCheckBox::clicked, [this](bool val)
+		    { m_gui_settings->SetValue(gui::gs_lockMouseFs, val); });
```

Comment:
Please format this as seen in other places

Reply:
This is from git clang-format, I will undo this